backend/apid/handlers/list_v3.go

```diff
@@ -0,0 +1,19 @@
+package handlers
+
+import (
+	"context"
+
+	corev2 "github.com/sensu/sensu-go/api/core/v2"
+	corev3 "github.com/sensu/sensu-go/api/core/v3"
+	"github.com/sensu/sensu-go/backend/store"
+	storev2 "github.com/sensu/sensu-go/backend/store/v2"
+)
+
+func (h Handlers) ListV3Resources(ctx context.Context, pred *store.SelectionPredicate) ([]corev3.Resource, error) {
+	req := storev2.NewResourceRequest(ctx, corev2.ContextNamespace(ctx), "", h.V3Resource.StoreName())
+	list, err := h.StoreV2.List(req, pred)
+	if err != nil {
+		return nil, err
+	}
+	return list.Unwrap()
+}
```

Comment:
For reference, listing resources will require a bit more work later because we need to support something equivalent to our [current lister](https://github.com/sensu/sensu-go/blob/main/backend/apid/routers/lister.go); not sure if this is something you intended to do in this PR or in a subsequent one, so just to make sure we don't forget about this!